packages/slate-editor/src/modules/editor-v4-rich-formatting/components/Lists/ListItemTextElement.tsx

```diff
@@ -18,13 +18,13 @@ export const ListItemTextElement: FunctionComponent<Props> = ({
     element,
     ...props
 }) => (
-    <div
+    <span
```

Comment:
Made this an inline element to have it play well with `list-style-position: inside` with non-default alignment modes.